starters/express-typeorm-postgres/README.md

```diff
@@ -0,0 +1,205 @@
+- `cd` into your project directory and run `npm install`.
+- Make sure you have docker & docker-compose installed on your machine
+- Create a `.env` file and copy the contents of `.env.example` into it.
+- Run `npm run infrastructure:start` to start the database and the redis instances
+- Run `npm run dev` to start the development server.
+- Open your browser to `http://localhost:3333/docs` to see the API documentation with the existing endpoints.
+
+## Commands
+
+- `npm run infrastructure:start` - Starts up a postgres database and two redis instances for caching
+- `npm run infrastructure:stop` - Stops the running database and redis docker containers.
+- `npm run db:seed` - Allows you to seed the database (See the Seeding section)
+- `npm run dev` - Starts the development server (Needs a running infrastructure first)
+- `npm run build` - Builds the app.
+- `npm start` - Starts the built app. (Needs a running infrastructure first)
+- `npm test` - Runs the unit tests.
+- `npm run lint` - Runs ESLint on the project.
+- `npm run format` - Formats code for the entire project
+- `npm run generate:schema`: - Generates the API schema types into the `src/interfaces/schema.ts` file
+
+## Example Controllers
+
+The starter contains an example CRUD implementation for technologies. You can find the controller and its handlers under the `/src/modules/technology/` folder.
+
+The handlers have caching enabled using the [cachified](https://www.npmjs.com/package/cachified) package. It uses redis under the hood. For more information on these endpoints, see the code, or check out the `localhost:3333/docs` after you start up your development server.
+
+## Database and Redis
+
+In order to start up your API in dev mode with an active database connection, please follow the following steps:
+
+1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
+2. run `npm run infrastructure:start`
+3. run `npm run dev`
+
+The above steps will make sure your API connects to the database and redis instances that gets started up with docker. When you finish work, run `npm run infrastructure:stop` to stop your database and redis containers.
+
+### Seeding
+
+In the `src/db/run-seeders.ts` file, we provide a script to seed the database with intial values, using TypeOrm. Under the `src/db/seeding` folder, you can find the `TechnologySeeder` class, that seeds values into the database as an example.
+
+In order to seed the database, you need to do the following steps:
+
+1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
+2. run `npm run infrastructure:start`
+3. run `npm run db:seed`
+
+### Reset infrastructure
+
+If you for some reason need to clear the contents of your database and you want to reinitialise it, delete the `misc/pg_data` folder and delete the postgres docker container. After that the next `infrastructure:start` command will start up as it would the first time.
```

Comment:
> delete the postgres docker container

What do you think about writing the command necessary for this?

Reply:
I don't know in advance what will the name of the containers be, because that depends on the container folder AFAIK